logic/atrium-logic/src/generated/commonMain/ch/tutteli/atrium/logic/iterableLike.kt

```diff
@@ -45,6 +45,9 @@ fun <T : IterableLike, E : Comparable<E>> AssertionContainer<T>.max(converter: (
 
 fun <T : IterableLike, E> AssertionContainer<T>.containsNoDuplicates(converter: (T) -> Iterable<E>): Assertion = impl.containsNoDuplicates(this, converter)
 
+fun <T : IterableLike, E : Comparable<E>> AssertionContainer<T>.last(converter: (T) -> Iterable<E>): FeatureExtractorBuilder.ExecutionStep<T, E> = impl.last(this, converter)
```

Comment:
I could not manage to get rid of `E : Comparable<E>`. Need help with it.

Reply:
hm. `last` should not require comparable, what error do you get when you just use `E`

Reply:
<img width="1072" alt="Screenshot 2024-03-10 at 23 21 33" src="https://github.com/robstoll/atrium/assets/5437573/635c3e88-639f-4e65-b894-a1c264850d4d">
Not sure if it's the best format to show an error :)

I've pushed a commit so you can have a look.